packer_templates/pkr-sources.pkr.hcl

```diff
@@ -274,4 +274,5 @@ source "vmware-iso" "vm" {
   winrm_timeout                  = var.winrm_timeout
   winrm_username                 = var.winrm_username
   vm_name                        = local.vm_name
+  cdrom_adapter_type             = var.vmware_cdrom_adapter_type
```

Comment:
set the disk adaptor type to point to the var as well and then delete lines 67-70 on pkr-sources file.